src/components/forms/newListingForm.jsx

```diff
@@ -115,6 +121,33 @@ class NewListingForm extends Component{
 
         );
     }
+    
+    submitForm = () => {
+        console.log("made it to submitform")
+        this.handleSubmit(this.state)
+        this.setState(this.initialState)
+    }
```

Comment:
May want to remove the console log print if it is no longer necessary. 
Very important to pass the character to the handle submit, before resetting the state! This allows the post to actually add the proper data. LGTM

Reply:
It has been changed thanks